src/entities/Flyer.ts

```diff
@@ -27,17 +31,13 @@ export class Flyer {
   @Property()
   location: string;
 
-  @Field()
-  @Property({ default: false })
-  nsfw: boolean;
```

Comment:
Seems like you removed nsfw field but didn't document this in the PR description, in the future remember to document all changes

Reply:
Ah thank you. I knew I was forgetting something.